test/com/stanimiriliev/javaobjects/sumator/SumatorTest.java

```diff
@@ -0,0 +1,37 @@
+package com.stanimiriliev.javaobjects.sumator;
+
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+/**
+ * @author Stanimir Iliev
+ * @since 29.07.2017
+ */
+class SumatorTest {
+    
+    @org.junit.jupiter.api.Test
+    void sum2() {
+        assertEquals(19, Sumator.sum("15", "4"));
+
+        try{
+            Sumator.sum("a", "b");
+            fail("The method should throw NumberFormatException");
+        }
+        catch(NumberFormatException e){
+            assertEquals(true, true);
```

Comment:
assertEquals(true, true);

that is wrong, because it does not check anything - it is always pass (true)

```java
int a = 12;
assertEquals(a, 12); - pass
assertNotEquals(a, 13); - pass

assertEquals(a, 1222); - not pass
```